gameyamlspiderandgenerator/util/spider.py

```diff
@@ -49,13 +44,10 @@ def get_status(url: AnyStr) -> SupportsInt:
 def download_file(url: AnyStr, save: AnyStr) -> SupportsInt:
     try:
         response = requests.get(url, allow_redirects=True)
-        if response.status_code == 200:
-            open(save, 'wb').write(response.content)
-            return 0
-        else:
+        if response.status_code != 200:
             return response.status_code
+        open(save, "wb").write(response.content)
```

Comment:
```suggestion
        with open(save, "wb") as f:
            f.write(response.content)
```

1. Use `with open()` instead of `open()`